src/games/strategy/net/ServerLoginHelper.java

```diff
@@ -59,9 +59,8 @@ public boolean canConnect() {
       }
       out.writeObject(challenge);
       out.flush();
-      @SuppressWarnings("rawtypes")
-      final Map credentials = (Map) in.readObject();//TODO rewrite in order to remove Suppressed Warning
-      final Set<Map.Entry<?,?>> entries = credentials.entrySet();
+      final Map<String, String> credentials = (Map<String, String>) in.readObject();//TODO rewrite in order to remove Suppressed Warning
```

Comment:
Well it's still supressed, becaus java can't check if this Object is the given generic type... But since there is no other way while using a ObjectInputStream I'll remove this comment